src/client/util/http.cc

```diff
@@ -380,7 +384,8 @@ bool HTTP::ProcessI2PResponse() {
       }
     }
   } else if (response_code
-             == static_cast<std::uint16_t>(ResponseCode::HTTP_NOT_MODIFIED)) {
+             == static_cast<std::uint16_t>(boost::network::http::basic_response<
+        boost::network::http::tags::http_server>::status_type::not_modified)) {
```

Comment:
These remaining two areas are also implied (I guess I should've been more literal). Can you please make the appropriate changes? Thanks.